SwissTransferFeatures/TransferDetailsView/EmptyStates/ExpiredTransferView.swift

```diff
@@ -16,18 +16,37 @@
  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+import STCore
 import STResources
 import SwiftUI
 import SwissTransferCoreUI
 
 struct ExpiredTransferView: View {
     @Environment(\.dismiss) private var dismiss
+    private let status: TransferStatus
+    private let transfer: TransferUi?
+
+    var subtitle: String {
+        switch status {
+        case .expiredDate:
+            return STResourcesStrings.Localizable.transferExpiredDescription
+        case .expiredDownloadQuota:
+            return STResourcesStrings.Localizable.transferExpiredLimitReachedDescription(Int(transfer?.downloadLimit ?? 0))
```

Comment:
Nide! Can you check if the plural form works? On the screenshot, it is written "1 downloadS"